docs/detections/add-exceptions.asciidoc

```diff
@@ -53,7 +53,12 @@ image::images/rule-exception-tab.png[Detail of rule exceptions tab]
 .. Click *Create shared exception list* -> *Create exception item*. 
 --
 
-. In the *Add rule exception* flyout, name the exception and add conditions that define the exception. When the exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria are met.
+. In the *Add rule exception* flyout, do the following: 
+
+.. Name the exception. 
+.. Add conditions that define the exception. When the exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria are met.
++
+NOTE: When you create a new exception from the alert's details flyout or from the Alert table, exception conditions are auto-populated with relevant alert data. A comment describing this action is also automatically added to the **Add comments** section.
```

Comment:
We could probably simplify this -- I get that naming each particular context adds specificity, but it gets the sentence off to a long, slow start, and users might not need to know every context (X and Y parts of the UI), just the general context (alerts).
```suggestion
NOTE: When you create a new exception from an alert, exception conditions are auto-populated with relevant alert data. A comment describing this action is also automatically added to the **Add comments** section.
```

Reply:
Yeah good point, especially when the necessary context is provided in the previous step. I'll commit this edit - thanks!